tests/test_neon.py

```diff
@@ -83,7 +83,53 @@ def simple_math_neon(n: size, x: R[n] @ DRAM, y: R[n] @ DRAM):  # pragma: no cov
         fn(None, n, x, y)
         assert np.allclose(x, expected)
 
+#@pytest.fixture
+def test_neon_vfmla():
```

Comment:
Can you put the expected scheduling output to `tests/golden/test_neon/test_neon_vfmla.txt` and take `golden` input, and compare the results?